src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -139,11 +138,11 @@ locationsAtPoint getHieFile IdeOptions{..} pos =
                 -- so we instead read the .hie files to get useful source spans.
```

Comment:
The comment that this mostly arises when the definition is in an external package probably needs updating. This is the case for DAML but I believe for Haskell, we will actually never find a `.hie` file for an external package.